fala/templates/generic-py-base.html

```diff
@@ -43,7 +44,7 @@
-            <image src="/assets/images/moj-logotype-crest.png" xlink:href="" class="moj-header__logotype-crest-fallback-image" width="40" height="40"></image>
+            <image src="/static/images/moj-logotype-crest.png" xlink:href="" class="moj-header__logotype-crest-fallback-image" width="40" height="40"></image>
```

Comment:
Why isn't this using the `static` helper method like you did on line 8 in this file?